pkg/cmd/render/render.go

```diff
@@ -154,6 +156,21 @@ func (r *renderOpts) Run() error {
 		mustReadTemplateFile(filepath.Join(r.generic.TemplatesDir, "config", "bootstrap-config-overrides.yaml")),
 		mustReadTemplateFile(filepath.Join(r.generic.TemplatesDir, "config", "config-overrides.yaml")),
 		&renderConfig,
+		map[string]resourcemerge.MergeFunc{
+			"apiServerArguments.feature-gates": func(dst, src interface{}, currentPath string) (interface{}, error) {
+				dest, ok := dst.([]interface{})
+				if !ok {
+					v, _ := json.Marshal(dest)
+					return nil, fmt.Errorf("expected apiServerArguments.feature-gates to be a string slice, but found: %v", v)
+				}
+				source, ok := src.([]interface{})
+				if !ok {
+					v, _ := json.Marshal(source)
+					return nil, fmt.Errorf("expected apiServerArguments.feature-gates to be a string slice, but found: %v", v)
+				}
+				return append(dest, source...), nil // append, later feature gates override earlier
```

Comment:
I expected us to stop here.  Stomping will allow us to set one value when we observe configuration, but allow full flexibility if the user goes for unsupportedConfigOverrides.